test-cases/longevity/longevity-large-partition-200pk-4days.yaml

```diff
@@ -0,0 +1,75 @@
+             "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=15 -clustering-row-count=100000 -partition-offset=121 -clustering-row-size=uniform:3072..5120 -concurrency=100 -connection-count=100 -rows-per-request=10 -timeout=30s -iterations 1 -validate-data",
+             "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=15 -clustering-row-count=100000 -partition-offset=136 -clustering-row-size=uniform:3072..5120 -concurrency=100 -connection-count=100 -rows-per-request=10 -timeout=30s -iterations 1 -validate-data",
+             "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=15 -clustering-row-count=100000 -partition-offset=151 -clustering-row-size=uniform:3072..5120 -concurrency=100 -connection-count=100 -rows-per-request=10 -timeout=30s -iterations 1 -validate-data",
+             "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=42 -clustering-row-count=100000 -partition-offset=166 -clustering-row-size=uniform:3072..5120 -concurrency=100 -connection-count=100 -rows-per-request=10 -timeout=30s -iterations 1 -validate-data",
+             "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=43 -clustering-row-count=100000 -partition-offset=208 -clustering-row-size=uniform:3072..5120 -concurrency=100 -connection-count=100 -rows-per-request=10 -timeout=30s -iterations 1 -validate-data"
+            ]
+
+stress_read_cmd: [
+                  "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=1000 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -partition-offset=1001 -concurrency=10 -connection-count=10 -rows-per-request=10 -iterations=10 -timeout=30s",
+                  "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=250 -clustering-row-count=100000 -clustering-row-size=uniform:3072..5120 -concurrency=10 -connection-count=10 -rows-per-request=10 -iterations=10 -timeout=30s -validate-data",
+                  "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=1000 -clustering-row-count=100000 -clustering-row-size=uniform:3072..5120 -rows-per-request=10 -timeout=30s -concurrency=100 -connection-count=100 -duration=6420m -validate-data",
+                  "scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=1000 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -rows-per-request=10 -timeout=30s -partition-offset=1001 -concurrency=100 -connection-count=100 -duration=6420m"
+                 ]
+
+n_db_nodes: 5
+n_loaders: 4
+n_monitor_nodes: 1
+
+round_robin: true
+
+instance_type_db: 'i3en.3xlarge'
+
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 30
+
+user_prefix: 'longevity-large-partitions-4d'
+
+space_node_threshold: 644245094
```

Comment:
Do we really use it? if not remove, if yes what is the default and why this value?

Reply:
It used when ```nemesis_during_prepare=true``` (that is default value). This value has come from original yaml. I think we need that. Need to wait for some prefill before start the nemesis.

Reply:
ok